### PR TITLE
Create new branch

### PR DESCRIPTION
**Pull request by NutaD0**
* Added new branch features
* Added some changes

### CODE REVIEW

**Comment by 1MPULSEONE** [comment]
All information in commits is written in the present tense (it's usually that way =) )

File: index.html [hunk 1]
```diff
@@ -9,14 +9,14 @@
     </head>
     <body>
 
-        <div class="background-header"></div>
-        <div><p class="site-name">Rest in Ocean</p></div>
-        <div class="text-bottom-of-text">
-            <h1>Text</h1>
-            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Excepturi saepe sunt delectus nobis, exercitationem aliquid ea hic placeat? Facere, qui.</p>
+        <div class="background-header">
+            <p class="site-name">Rest in Ocean</p>
```
Comment (1MPULSEONE) [inline index.html:13]: Also semantics suffers in the code, read about it. 
At the very least, I would use "header" and "h1" tags 
 https://developer.mozilla.org/ru/docs/Web/HTML/Element/header

File: index.html [hunk 2]
```diff
@@ -9,14 +9,14 @@
     </head>
     <body>
```
Comment (1MPULSEONE) [inline index.html:10]: Use formatter later

File: styles/style.css [hunk 3]
```diff
@@ -8,55 +8,42 @@ html, body {
 }
 
 .background-header{
-    position: absolute;
-    background-image: url("./background-image.png");
+    position: relative;
```
Comment (1MPULSEONE) [inline styles/style.css:11]: I think there`s no point of this.

File: index.html [hunk 4]
```diff
@@ -9,14 +9,14 @@
...
 
-        <div class="background-header"></div>
-        <div><p class="site-name">Rest in Ocean</p></div>
-        <div class="text-bottom-of-text">
-            <h1>Text</h1>
-            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Excepturi saepe sunt delectus nobis, exercitationem aliquid ea hic placeat? Facere, qui.</p>
+        <div class="background-header">
```
Comment (1MPULSEONE) [inline index.html:12]: Why header contains welcome block?

File: styles/style.css [hunk 5]
```diff
@@ -8,55 +8,42 @@ html, body {
...
-    background-position-y: 0px;
-    height: 100vh;
     width: 100%;
-    top: -10px;
+    height: 100vh;
     z-index: -1;
 }
 
-.site-name{
+.site-name {
```
Comment (1MPULSEONE) [inline styles/style.css:21]: Good example of bad naming

File: styles/style.css [hunk 6]
```diff
@@ -8,55 +8,42 @@ html, body {
...
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    padding: 0.34em;
```
Comment (1MPULSEONE) [inline styles/style.css:25]: It's good that you use em, but unless the layout requires it, I usually round up the values for lighter values (0.35 in your case)

File: styles/style.css [hunk 7]
```diff
@@ -8,55 +8,42 @@ html, body {
...
     color: white;
     font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```
Comment (1MPULSEONE) [inline styles/style.css:27]: It`s better to use variables

File: styles/style.css [hunk 8]
```diff
@@ -8,55 +8,42 @@ html, body {
...
     z-index: -1;
 }
 
-.site-name{
+.site-name {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    padding: 0.34em;
     color: white;
```
Comment (1MPULSEONE) [inline styles/style.css:26]: Also think about variables

File: styles/style.css [hunk 9]
```diff
@@ -8,55 +8,42 @@ html, body {
 }
 
 .background-header{
-    position: absolute;
-    background-image: url("./background-image.png");
+    position: relative;
+    margin: 0;
+    padding: 0;
+    background-image: url(./background-image.png);
```
Comment (1MPULSEONE) [inline styles/style.css:14]: Maybe move images to some folder?

File: styles/style.css [hunk 10]
```diff
@@ -8,55 +8,42 @@ html, body {
...
-    font-size: 40px;
     color: white;
-    text-shadow: 2px 2px 3px rgba(0, 0, 0, 0.356);
-    z-index: -1;
+    font-size: 2rem;
+    padding-top: 5em;
+    padding-bottom: 9.3em;
+    padding-right: 5em;
+    margin-top: -2em;
+    margin-right: 29.7em;
```
Comment (1MPULSEONE) [inline styles/style.css:44]: Try to avoid any use of margin

Comment (1MPULSEONE) [reply styles/style.css:44]: (Also strange values, make them easier to read)

**Review by 1MPULSEONE — CHANGES_REQUESTED** [review]

File: styles/style.css (hunk 6 above)
Comment (NutaD0) [reply styles/style.css:25]: I just wanted to do everything really mega accurately

File: styles/style.css (hunk 7 above)
Comment (NutaD0) [reply styles/style.css:27]: how can i use variables?

File: styles/style.css (hunk 9 above)
Comment (NutaD0) [reply styles/style.css:14]: I'll have to do this

File: styles/style.css (hunk 10 above)
Comment (NutaD0) [reply styles/style.css:44]: I used this to raise or lower an HTML element, but I don’t know how else to do it

**Pull request merged**